src/status_im/contexts/wallet/wallet_connect/modals/common/footer/view.cljs

```diff
@@ -13,23 +13,32 @@
   (rf/dispatch [:wallet-connect/respond-current-session password]))
 
 (defn view
-  [{:keys [warning-label slide-button-text disabled?]} & children]
-  (let [{:keys [customization-color]} (rf/sub [:wallet-connect/current-request-account-details])]
-    [rn/view {:style style/content-container}
-     (into [rn/view
-            {:style style/data-items-container}]
-           children)
-     [rn/view {:style style/auth-container}
-      [standard-authentication/slide-button
-       {:size                :size-48
-        :track-text          slide-button-text
-        :disabled?           disabled?
-        :customization-color customization-color
-        :on-auth-success     on-auth-success
-        :auth-button-label   (i18n/label :t/confirm)}]]
-     [rn/view {:style style/warning-container}
-      [quo/text
-       {:size   :paragraph-2
-        :style  {:color colors/neutral-80-opa-70}
-        :weight :medium}
-       warning-label]]]))
+  [{:keys [warning-label slide-button-text error-text]} & children]
+  (let [{:keys [customization-color]} (rf/sub [:wallet-connect/current-request-account-details])
+        network-status                (rf/sub [:network/status])
+        offline?                      (= network-status :offline)]
```

Comment:
you're right. This would apply to both subs here it seems. For the first one we may need a sub specific for the color. As for the network status, I remember we shouldn't use root subs inside components. I'll make a layer 2 sub for it that returns a boolean 👍. Thank you!

Reply:
done